.travis.yml

```diff
@@ -6,8 +6,8 @@ sudo: false
 perl6:
   - latest
 install:
-  - rakudobrew build-panda
-  - panda installdeps .
+  - rakudobrew build-zef
+  - zef --debug install .
```

Comment:
AFAIK Travis starts from a clean environment for every build so this should not matter, but I'd change it anyway.